test/spanish-hyphens2.txt

```diff
@@ -0,0 +1,8635 @@
+# word word_hyphenated number_syllables accented_vowel_position
+# number_syllables and accented_vowel_position starts counting from 1, i.e., 'gustaría gus-ta-rí-a 3 7' is 3 syllables and the accented vowel is the seventh.
```

Comment:
I'd prefer if this file followed what the library does, i.e., using 0-based counts for both.

Reply:
Done